MatrixSDK/Threads/MXThreadingService.swift

```diff
@@ -100,35 +107,57 @@ public class MXThreadingService: NSObject {
         return result
     }
     
+    /// Get threads in a room
+    /// - Parameter roomId: room identifier
+    /// - Returns: thread list in given room
+    public func threads(inRoom roomId: String) -> [MXThread] {
+        //  sort threads so that the newer is the first
+        return Array(threads.values).filter({ $0.roomId == roomId }).sorted(by: <)
```

Comment:
I'm not sure how to achieve this (and it's probably unnecessary early optimisation), but it would be kind of nice if this had a `.lazy` to improve performance in the second `filter` in `participatedThreads`.

Reply:
Ah that doesn't seem trivial to me, let's handle it later.